docs/user-guide/aws-examples/eks.rst

```diff
@@ -0,0 +1,229 @@
+----------------
+
+* EKS Cluster:
+    * `Dask Operator <https://kubernetes.dask.org/en/latest/installing.html>`__
+    * If self managed node group is created with ubuntu worker nodes then install GPU operator. When setting up a self-managed node group with Ubuntu worker nodes in Amazon EKS, it's advantageous to install the GPU Operator. The GPU Operator is highly recommended as it simplifies the deployment and management of NVIDIA GPU resources within Kubernetes clusters. This operator automates the installation of NVIDIA drivers, integrates with container runtimes like containerd through the NVIDIA Container Toolkit, manages device plugins, and provides monitoring capabilities.
+        `GPU operator <https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html>`__
+    * If EKS managed node group is created with Amazon Linux 2 worker nodes then install Nvidia device plugin. This approach has a limitation, the pre-installed NVIDIA GPU driver version and NVIDIA container runtime version lags the release schedule from NVIDIA requires you to assume the responsibility ofr upgrading the NVIDIA devicw plugin version.
+        `Nvidia Device Plugin installation <https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html>`__
+     For more details, please refer `NVIDIA GPU Operator with Amazon EKS <https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/amazon-eks.html>`__
+* Storage:
+    * `EFS for EKS <https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/docs/efs-create-filesystem.md>`__ (setup by Kubernetes cluster admin)
+
+Create a Storage Class for AWS EFS
+----------------------------------
+
+.. code-block:: yaml
+
+   cat <<EOF | kubectl apply -f -
+   kind: StorageClass
+   apiVersion: storage.k8s.io/v1
+   metadata:
+     name: efs-sc
+   provisioner: efs.csi.aws.com
+   parameters:
+     provisioningMode: efs-ap
+     fileSystemId: ${FileSystemId}  # Replace with your actual FileSystemId
+     directoryPerms: "700"
+   EOF
+
+In the above YAML:
+
+- Replace `${FileSystemId}` with the actual EFS FileSystemId from AWS.
+- This definition sets up a StorageClass named `efs-sc` that provisions PersistentVolumes using the AWS EFS CSI Driver.
+
+
+PVC Definition
+--------------------------------
+Now, we can use the storage class created in the previous step to create a PVC.
+
+.. code-block:: yaml
+
+   cat <<EOF | kubectl apply -f -
+   apiVersion: v1
+   kind: PersistentVolumeClaim
+   metadata:
+     name: nemo-workspace
+   spec:
+     accessModes:
+       - ReadWriteMany
+     storageClassName: efs-sc
+     resources:
+       requests:
+         storage: 150Gi
+   EOF
+
+This PVC requests 150GiB of storage with ReadWriteMany access mode from the efs-sc StorageClass.
+
+Dask cluster creation:
+----------------------
+
+Please refer index.rst for instructions on creating a Docker secret to utilize the NeMo image and upload data to the PVC created in the previous step.
```

Comment:
Can you fix this link and also take into account my other feedback about calling out the relevant sections?